Marlin/src/feature/power_loss_recovery.cpp

```diff
@@ -315,15 +315,18 @@ void PrintJobRecovery::resume() {
     memcpy(&mixer.gradient, &info.gradient, sizeof(info.gradient));
   #endif
 
-  // Restore Z (plus raise) and E positions with G92.0
+  // home  
+  gcode.process_subcommands_now_P(PSTR("G28"));
```

Comment:
Note: You can't home with `G28` when resuming a print. You will crush the object on the bed.